internal/webhooks/v1beta1/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2021 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2025 The Metal3 Authors.
```

Reply:
We cannot change the copyright just because we move the file. If it was initially copied from somewhere with Kubernetes Authors, it should stay that way.

Reply:
Actually I did some googling and I take back my statement, even if it is skeleton file, we should not remove the original copyright as @lentzi90 pointed, but rather we are ok to add ours (if we want to) in the next line i.e. Sorry for the confusion and overwork created by my comment previously. 

```
Copyright 2021 The Kubernetes Authors.
Copyright 2025 The Metal3 Authors.
```

Reply:
I will fix that

Reply:
fixed